<launch>

    <arg name="world" default="empty"/>
    <arg name="robot_name" default="marv"/>

    <!-- Start bridges for flipper controllers -->
    <include file="$(dirname)/flipper_control.launch">
        <arg name="robot_name" value="$(arg robot_name)"/>
        <arg name="joint_prefix" value="front_left"/>
    </include>

    <include file="$(dirname)/flipper_control.launch">
        <arg name="robot_name" value="$(arg robot_name)"/>
        <arg name="joint_prefix" value="front_right"/>
    </include>

    <include file="$(dirname)/flipper_control.launch">
        <arg name="robot_name" value="$(arg robot_name)"/>
        <arg name="joint_prefix" value="rear_left"/>
    </include>

    <include file="$(dirname)/flipper_control.launch">
        <arg name="robot_name" value="$(arg robot_name)"/>
        <arg name="joint_prefix" value="rear_right"/>
    </include>
</launch>
